src/python/grpcio/grpc/_channel.py

```diff
@@ -85,7 +85,7 @@ def _deadline(timeout):
 
 
 def _unknown_code_details(unknown_cygrpc_code, details):
-  return b'Server sent unknown code {} and details "{}"'.format(
+  return 'Server sent unknown code {} and details "{}"'.format(
```

Comment:
Just to make sure: this string *never* goes back down to C core, right? It's sent straight up to the user?